edb/schema/pointers.py

```diff
@@ -482,13 +470,13 @@ def merge_targets(
                 # conflict.
                 vnp = ptr.get_verbosename(schema, with_parent=True)
                 vn = ptr.get_verbosename(schema)
+                t1_vn = t1.get_verbosename(schema)
                 t2_vn = t2.get_verbosename(schema)
                 raise errors.SchemaError(
                     f'cannot redefine {vnp} as {t2_vn}',
-                    details=(
-                        f'{vn} targets {t2_vn} that is not related '
-                        f'to a type found in this link in the parent type: '
-                        f'{t1.get_displayname(schema)!r}.'))
+                    details=f'{vn} is defined as {t1_vn} in a parent type, '
```

Comment:
"in the parent type"? Also, I assume that `t1_vn` will include the parent type name, right?

Reply:
No it won't

Reply:
And also, we are talking about _a_ parent type, because there may be several.

Reply:
Can we include the actual name of the parent to the error message? Why can't we be specific here?

Reply:
That would require some refactoring as we aren't passing the necessary context through the call stack. I can follow up with a cleanup later.